src/bundles.sh

```diff
@@ -0,0 +1,18 @@
+export CONDUCTR_IP=$(hostname -i)
+
+conduct load conductr-haproxy
+conduct run conductr-haproxy --scale $NODE_COUNT
+
+sudo service haproxy restart
+
+sleep 5
+
+ES_COUNT=$(( NODE_COUNT < 3 ? NODE_COUNT : 3 ))
+
+conduct load conductr-elasticsearch
+conduct run conductr-elasticsearch --scale $ES_COUNT
+
+sleep 5
```

Comment:
Maybe we don't need it here. I think I needed the first one so I guess I just added this one too. The first was maybe only needed because of the haproxy restart so I guess we could take this out. In the past when I've used ConductR in a script like this I've found things go smoother with little delays like this between commands working with different bundles but that was a while ago and maybe it was all in my head. If you don't think it's necessary then it can be removed.